doc/dev/tests.md

```diff
@@ -128,8 +126,11 @@ A quick description of the five commands above:
 ## `devtools_testutils` Package
 The Azure SDK team has created some in house tools to help with easier testing. These additional tools are located in the `devtools_testutils` package that was installed with your `dev_requirements.txt`. In this package is the [`AzureTestCase`](https://github.com/Azure/azure-sdk-for-python/blob/main/tools/azure-sdk-tools/devtools_testutils/azure_testcase.py#L99-L350) object which every test case object should inherit from. This management object takes care of creating and scrubbing recordings to make sure secrets are not added to the recordings files (and subsequently to the git history) and authenticating clients for test methods.
```

Comment:
I'm wondering if we should move the sentences about test case objects inheriting from `AzureTestCase` to where we talk about "older SDK tests". Otherwise this guidance doesn't mix with test proxy. I can make the change if you agree

Reply:
My thought is that a lot of McCoy's test proxy stuff should go here.  @mccoyp do you agree?